_episodes/06-conclusion.md

```diff
@@ -40,7 +40,7 @@ There are many local opportunities at the University of Michigan or around the A
 
 * [Software Carpentry Workshops](https://umswc.github.io/workshops/)
     - We're always looking for more helpers & instructors!
-* [CoderSpaces](https://datascience.isr.umich.edu/), [CoderSpaces Slack Space](https://join.slack.com/t/umcoderspaces/signup)
+* [CoderSpaces](https://datascience.isr.umich.edu/)
```

Comment:
maybe use this link instead?
https://datascience.isr.umich.edu/events/coderspaces/